service/src/main/java/uk/gov/hmcts/reform/rd/model/User.java

```diff
@@ -0,0 +1,14 @@
+package uk.gov.hmcts.reform.rd.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class User {
+    private String userIdentifier;
```

Comment:
Do you need setter for that class?

Reply:
Yes, please see my comment in the FeignConfiguration, if time better would investigate a bit more.